src/App.jsx

```diff
@@ -1,13 +1,25 @@
-import { useState } from 'react'
-import './App.css'
+import React from 'react';
+import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
```

Comment:
React updated the BrowserRouter syntax, and our code currently has that provided in router.jsx and the <RouterProvider> tag. You can structure the additional paths you want to add in router.jsx

Feel free to reference the layout structure here!:
https://github.com/beefchief-code/group5.FSU.frontend/tree/main/src/layout